nbextensions/appCell2/widgets/configureTab.js

```diff
@@ -82,23 +83,20 @@ define([
 
     function factory(config) {
         var container,
-            widget,
-            workspaceInfo;
+            widget;
 
         function start(arg) {
             container = arg.node;
-            workspaceInfo = arg.workspaceInfo;
 
             return loadParamsWidget({
-                    node: container,
-                    workspaceInfo: arg.workspaceInfo,
-                    appSpec: arg.appSpec,
-                    parameters: arg.parameters
+                node: container,
+                workspaceInfo: arg.workspaceInfo,
+                appSpec: arg.appSpec,
+                parameters: arg.parameters
 
-                })
-                .then(function(result) {
-                    widget = result;
-                });
+            }).then(function(result) {
```

Comment:
These were just edits to make eslint happy. I did them while looking at this file.